Tinylog/View Controllers/Lists/AddListViewController.swift

```diff
@@ -102,20 +115,20 @@
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return 1
+        1
     }
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return 44.0
+        44.0
     }
-
+    
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell: TextFieldCell = TextFieldCell(style: .default, reuseIdentifier: "CellIdentifier")
-        configureCell(cell, indexPath: indexPath)
+        let cell: TextFieldCell = tableView.dequeue(for: indexPath)
+        configureCell(cell, atIndexPath: indexPath)
         return cell
     }
-
-    private func configureCell(_ cell: TextFieldCell, indexPath: IndexPath) {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/03eace86b303bd47c13e99b11bd2b1214da5c26b/Tinylog/View Controllers/Lists/AddListViewController.swift#L130"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AddListViewController.swift:130</code></td>
    </tr>
  </tbody>
</table>